src/Generator/FileGenerator.php

```diff
@@ -408,6 +420,25 @@ public function getBody()
         return $this->body;
     }
 
+    public function setDeclares(array $declares)
+    {
+        foreach ($declares as $declare) {
+            if (! $declare instanceof DeclareStatement) {
+                throw new InvalidArgumentException(sprintf(
+                    '%s is expecting an array of %s\Declare objects',
```

Comment:
error message should be updated: DeclareStatement

Reply:
Will replace this will class contant itself `DeclareStatement::class` as it is more secure to refactoring.